docs/build-with-javascript.md

```diff
@@ -0,0 +1,558 @@
+| /cmd         | applications executables            |
+| /cmd/migrate | install database schema             |
+| /cmd/sensor  | Simulates the pet's collar          |
+| /config      | database configuration              |
+| /db          | database handlers (gocql/x)         |
+| /db/cql      | database schema                     |
+| /handler     | REST API handlers                   |
+| /model       | application models and ORM metadata |
+
+## Quick Start
+
+Prerequisites:
+
+- [NodeJS](https://nodejs.org/en/) tested with v17.0.1
+- [NPM](https://www.npmjs.com/) tested with v8.1.0
+- [docker](https://www.docker.com/) (not required if you use Scylla Cloud)
+- [docker-compose](https://docs.docker.com/compose/) (not required if you use Scylla Cloud)
+
+
+Clone the repository and move to `javascript` directory:
+```
+git clone git@github.com:scylladb/care-pet.git
+cd javascript
+```
+
+Make sure to install all NodeJS dependencies with:
+
+    $ npm install
+    
+### Use ScyllaDB on your local machine
+
+To run a local ScyllaDB cluster consisting of three nodes with
+the help of `docker` and `docker-compose` execute:
+
+    $ docker-compose up -d
+
+Docker-compose will spin up three nodes: `carepet-scylla1`, `carepet-scylla2`
+and `carepet-scylla3`. You can access them with the `docker` command.
+
+Execute the following nodetool command:
+
+    $ docker exec -it carepet-scylla1 nodetool status
+
+### Migrate
+
+#### Using ScyllaDB on your local machine
+
+Once all the nodes are in UN - Up Normal status, run the below commands:
+
+The below command allows you to get node IP address:
+
+```
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1
+```
+
+The run the following commands to execute the migrate main function.
+
+```
+NODE1=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1)
+npm run migrate -- --hosts $NODE1
+```
+#### Using Scylla Cloud
+
+If you are using Scylla Cloud, use the the following command:
+
+```
+npm run migrate -- --hosts [NODE-IP] --username [USERNAME] --password[PASSWORD]
+```
+
+Replace the NODE-IP, USERNAME and PASSWORD with the information provided in your cluster on https://cloud.scylladb.com.
+
+expected output:
+```
+2020/08/06 16:43:01 Bootstrap database...
+2020/08/06 16:43:13 Keyspace metadata = {Name:carepet DurableWrites:true StrategyClass:org.apache.cassandra.locator.NetworkTopologyStrategy StrategyOptions:map[datacenter1:3] Tables:map[gocqlx_migrate:0xc00016ca80 measurement:0xc00016cbb0 owner:0xc00016cce0 pet:0xc00016ce10 sensor:0xc00016cf40 sensor_avg:0xc00016d070] Functions:map[] Aggregates:map[] Types:map[] Indexes:map[] Views:map[]}
+```
+
+You can check the database structure with:
+
+#### Using ScyllaDB on local machine
+```
+docker exec -it carepet-scylla1 cqlsh
+```
+
+#### Using Scylla Cloud
+
+`docker run -it --rm --entrypoint cqlsh scylladb/scylla -u [USERNAME] -p [PASSOWRD] [NODE-IP]`
```

Comment:
It looks like these two headings should be preceded by a more generic heading and an explanation of what the commands do.
It is unclear also because there are identical headings above (Using ScyllaDB on your local machine, Using Scylla Cloud).